dependencyManagement/build.gradle.kts

```diff
@@ -96,7 +97,9 @@ val DEPENDENCIES = listOf(
   "commons-logging:commons-logging:1.3.0",
   "commons-validator:commons-validator:1.8.0",
   "io.netty:netty:3.10.6.Final",
-  "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:1.33.0-alpha",
+  "io.opentelemetry.contrib:opentelemetry-aws-resources:${otelContribVersion}",
+  "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:${otelContribVersion}",
+  "io.opentelemetry.contrib:opentelemetry-gcp-resources:${otelContribVersion}",
```

Comment:
if these changes aren't needed for the manifest resource detector can you move them into a separate PR?

Reply:
That was a rebase gone wrong - thanks for spotting :smile: